uqcsbot/scripts/crisis.py

```diff
@@ -13,8 +13,8 @@
 @bot.on_command("mentalhealth")
 @bot.on_command("emergency")
 def handle_crisis(command: Command):
-    '''
+    """
     `!crisis`, `!mentalhealth` or `emergency` - Get a list of emergency resources.
```

Comment:
Not your fault but emergency should be prefixed with !

Reply:
Fixed.